src/server/attr_recov_db.c

```diff
@@ -113,7 +106,7 @@ compare_obj_hash(void *qs, int len, void *oldhash)
  *
  */
 int 
-encode_single_attr_db(struct attribute_def *padef, struct attribute *pattr, pbs_db_attr_list_t *db_attr_list)
+encode_single_attr_db(struct attribute_def *padef, struct attribute *pattr, int index, pbs_db_attr_list_t *db_attr_list)
```

Comment:
Looks like we don't need `index` arg anymore?

Reply:
yeah i had it to save the index in the database as well, however in our updated multiserver approach since we will not be loading from the database that often - that optimization does not make sense, so i will remove this